src/components/structures/RoomView.js

```diff
@@ -91,6 +90,7 @@ module.exports = React.createClass({
         return {
             room: null,
             roomId: null,
+            userId: null,
```

Comment:
Hmm... interesting ... it looks like the state was being used in my commit - https://github.com/matrix-org/matrix-react-sdk/commit/dc4f321707da8d7431c857ab6abf8240e671fb31. I'm not sure how this got changed, but I've removed it from the state in preference for accessing the value directly, as you pointed out in :1619